core/orch_test.go

```diff
@@ -387,13 +417,13 @@ func TestTranscoderManagerTranscoding(t *testing.T) {
 
 	// fatal error should retry and remove from list
 	s.SendError = fmt.Errorf("SendError")
-	_, err = m.Transcode(&SegTranscodingMetadata{})
+	_, err = m.Transcode(&SegTranscodingMetadata{AuthToken: &net.AuthToken{SessionId: testSessionId}})
 	assert.True(wgWait(wg)) // should disconnect manager
 	assert.NotNil(err)
-	assert.Equal(err.Error(), "No transcoders available")
-	_, err = m.Transcode(&SegTranscodingMetadata{}) // need second try to remove from remoteTranscoders
+	assert.Equal(err.Error(), "no transcoders available")
```

Comment:
Just a suggestion: there is also assert.EqualError(error, expected message) 

We can also use the variable we created for this specific error : ErrNoTranscodersAvailable